src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import { email, validatePassCoincidence } from 'utils/inputValidations';
+
+import wLogo from '../../assets/wLogo.png';
+
+import 'scss/layout.scss';
+import 'scss/components.scss';
+import 'scss/margins.scss';
+import styles from './styles.module.scss';
+
+interface Inputs {
+  name: string;
+  lastName: string;
+  email: string;
+  password: string;
+  confirmPassword: string;
+}
+
+function SignUp() {
+  const { register, errors, getValues, setError } = useForm<Inputs>();
+  //   const onSubmit = (data: Inputs) => console.log(data);
+  const onBlur = (): void => {
```

Comment:
It could be done using useForm and useForm with reValidateMode: 'onChange' moving this validation to register ref

See more; https://react-hook-form.com/api